src/articles/list.tsx

```diff
@@ -64,10 +69,43 @@ export function ArticleCard(props: ArticleCardProps, renderer: RendererLike<Node
 export interface ArticleListProps {
   articles: Source<Article[]>;
   title?: string;
+  isLoading?: State<boolean>;
   pick?: (article: Article) => void;
+  loadMore?: () => void;
 }
 
-export function ArticleList(props: ArticleListProps, renderer: RendererLike<Node>) {
+export function ArticleList(
+  this: LiveComponentThis,
+  props: ArticleListProps,
+  renderer: RendererLike<Node>
+) {
+  const handleScroll = () => {
```

Comment:
this can be handled neatly with [`fromEvent()`](https://loreanvictor.github.io/callbag-common/factory/from-event) and [`this.track()`](https://loreanvictor.github.io/callbag-jsx/components/tracking), instead of manually registering and removing a scroll listener to the window.

Reply:
Done. Very nice!